docs/making_transfers.md

```diff
@@ -1,171 +1,243 @@
----
-title: Transfers

```

Comment:
Looks like a mistake: the documentation page about transfer has been replaced by documentation about the ledger signer.

Reply:
replaced the page with transfers content and updated it